test/functional/api/vim_spec.lua

```diff
@@ -1148,6 +1148,13 @@ describe('API', function()
     before_each(function()
       meths.set_option('isident', '')
     end)
+
+    local it
+    if (helpers.isCI('appveyor') and os.getenv('CONFIGURATION') == 'MSVC_32') then
+      -- 'works with &opt' is flaky on MSVC_32, but not easy to skip alone.  #10241
+      it = pending
```

Comment:
Let's not use this pattern. It will cause many "SKIP" messages, we only want 1 message.

Reply:
That will not work here.. the tests are run in/via a different file - and you would also skip "everything below".

To only disable the flaky test (https://github.com/blueyed/neovim/blob/055bfca30b9d58c68bcca74977ce590a0c052d0c/test/unit/viml/expressions/parser_tests.lua#L6329) we would need to pass through "pending" as-is, and use it for it then.  Would require to change the interface (but might be ok - but I do not think it's worth the extra effort, for something that likely indicates a bug anyway.. :/)).